magento/pwa-studio
# [Doc] revise the update footer tutorial

## Description

Revise and publish the Update Footer tutorial contributed by @rossmc 

## Related Issue

Closes PWA-302

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. navigate to the pwa-devdocs directory: `cd pwa-devdocs`
1. Run linter on file: `yarn lint src/tutorials/pwa-studio-fundamentals/modify-site-footer`
1. Verify no linting issues
1. Run preview server: `yarn develop`
1. Navigate to HTML preview site and go to the Tutorials section
1. Verify item appears on left navigation and links to the correct page
1. Verify page renders and content is accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
